.github/workflows/publish.yml

```diff
@@ -0,0 +1,95 @@
+name: Test & Build & Publish to Pypi
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  # test:
+  #   if: startsWith(github.event.head_commit.message, 'chore(release)')
+  #   uses: "./.github/workflows/test.yml"
+
+  build:
+    # needs: 
+    # - test
+    if: startsWith(github.event.head_commit.message, 'chore(release)')
+    runs-on: ubuntu-latest
+    steps:  
+    - name: Checkout code
+      uses: actions/checkout@v4
+    
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.x'
```

Comment:
I'm considering freezing to the most recent supported version (`3.12`?), so we could have full control over when we update.

I had an experience in other unrelated projects with GitHub Actions suddenly breaking after something had been updated behind the scenes and losing a significant time to fix that instead of the initial plan of working on new features. Since then I have adopted the careful approach of freezing whatever I can in CI/CD.

WDYT?

Reply:
Sure, make sense!